readme.md

```diff
@@ -102,7 +119,7 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+> If validations fail, the first line will fail silently, while the second line will throw a fatal error.
 
```

Comment:
Disregard part of my earlier comment - here's a Rails-y question!

I think we'd learn more about where the students are at if we quiz them on ActiveRecord on a more fundamental level. How about asking them to list the Rails equivalents of some Mongoose methods (and/or vice-versa)?